app/helpers/browse_helper.rb

```diff
@@ -86,7 +86,7 @@ def sidebar_classic_pagination(pages, page_param)
     max_width_for_default_padding = 35
 
     width = 0
-    pagination_items(pages, {}).each do |body|
+    pagination_items(pages, {}).each do |(body)|
```

Comment:
Just use `|body, _|` instead if you want to destructure it without an unused variable warning.

Reply:
Rubocop is going to complain then that I'm probably iterating over a hash and that I should use each_key instead.

Reply:
Body is an array, like `["3", "active"]`, so applying `each_key` here might be a bit confusing (why is the first array element a key?). On the other hand, I'm not clear how `(body)` works.

In my proposal I left this part as is, and opted to access the array by element index instead:
`width += body.first.length` or `width += body[0].length`

By the way, I was wondering a bit why pagination_items returns an array of arrays, instead an array of hashes, where the purpose of each attribute would be a bit clearer. That's maybe something for another issue.

Reply:
Using `each_key` won't work anyway as it's only defined on hashes. Dropping the `each` and making `pagination_items` yield the items directly instead of building a list and returning it does then allow the syntax I wanted without warnings but as you say maybe these should be hashes if that allows us to get rid of the `page_or_class` duality thing.